packages/obj-logger/src/utils/pretty-print.ts

```diff
@@ -0,0 +1,17 @@
+import { COLORS } from "./colors";
+import { LogEntry, LogLevel } from "@scramjet/types";
+
+const COLOR_MAP: { [ key: string]: string } = {
+    TRACE: COLORS.Dim,
+    DEBUG: COLORS.FgBlue,
+    INFO: COLORS.FgCyan,
+    WARN: COLORS.FgYellow,
+    ERROR: COLORS.FgRed,
+    FATAL: COLORS.FgMagenta
+};
+
+const getLevelColor = (level: LogLevel = "INFO") => `${COLOR_MAP[level]}${level?.padEnd(5)}${COLORS.Reset}`;
+
+export const prettyPrint = (obj: LogEntry) => {
```

Comment:
Couldn't find any references in the code? Shouldn't this be in cli?

Reply:
this is used in host https://github.com/scramjetorg/transform-hub/blob/feature/obj-logger/packages/host/src/lib/host.ts#L120